ui/StatusQ/src/StatusQ/Core/StatusAssetSettings.qml

```diff
@@ -33,10 +33,14 @@ QtObject {
     property int bgBorderWidth: 0
 
     //image
-    property bool isImage: false
+    property bool isImage: isImgSrc(root.name)
     property int imgStatus
     property bool imgIsIdenticon: false
 
     // ring settings hints
     readonly property real ringPxSize: Math.max(1.5, root.width / 24.0)
+
+    function isImgSrc(name) {
+        return name.includes("data") || name.toLowerCase().startsWith("http://") || name.toLowerCase().startsWith("https://")
```

Comment:
```qml
return name.startsWith(Constants.dataImagePrefix) || name.toLowerCase().startsWith("http://") || name.toLowerCase().startsWith("https://")
```

Reply:
Good suggestion but can't use it, `Constants` are not part of `StatusQ`. I would need to introduce StatusQ's `Constants` and move it there but I don't think it should be in the scope of this PR.

Reply:
Ah right :/ Still even a hardcoded `startsWith("data:image")` would be better imho

Reply:
Agreed, improved